NextcloudTalk/ReferenceView.swift

```diff
@@ -1,25 +1,9 @@
 //
-// Copyright (c) 2022 Marcel Müller <marcel-mueller@gmx.de>
-//
-// Author Marcel Müller <marcel-mueller@gmx.de>
-//
-// GNU GPL version 3 or any later version
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+// SPDX-FileCopyrightText: 2022 Nextcloud GmbH and Nextcloud contributors
+// SPDX-License-Identifier: GPL-3.0-or-later
 //
 
-@objcMembers class ReferenceView: UIView {
+class ReferenceView: UIView {
```

Comment:
While unrelated to this PR, the change is fine, as we moved the ChatView over to Swift in the meantime.